src/Transpiler.ts

```diff
@@ -91,6 +92,8 @@ export abstract class LuaTranspiler {
         this.importCount = 0;
         this.sourceFile = sourceFile;
         this.isModule = tsHelper.isFileModule(sourceFile);
+        this.isStrict = options.alwaysStrict || options.strict
+            || (options.target && options.target > ts.ScriptTarget.ES5);
```

Comment:
Is ES5+ always strict? Or (if it's default true) did you forget to check if it is not explicitly set to false.

Reply:
Ok, I just looked this up in the ecma specs. In ES5, strict mode has to be explicitly enabled (with "use strict" which is what alwaysStrict produces). In ES6 (aka ES2015) and beyond, modules are always strict (I don't believe it can be disabled), but non-modules still require the explicit enabling. I'll update to match that logic.